### ISSUE TITLE
How openhab connected to DiyonXBee arduino device?

### ISSUE THREAD

**Issue by musthafapa** [juri8/diyonxbee-library#3]
I just discovered diyonxbee. I would like to know how openhab is connected to DiyonXBee arduino device? how one xbee device communicated to another xbee device

**Comment by juri8** [comment]
Hi, this currently doesn't happen. The microcontrollers on the XBees just use them transparently (so they just write to the serial output where the XBee is connected). By default, all that communication goes to the coordinator, which is connected to my openhab server. I decided to do it this way because it's a lot easier to maintain rules in openhab than it would be to update the microcontrollers. The openhab plugin uses the XBees in API mode, so it can send messages to a dedicated destination.

FYI: I recently tried to use my plugin in openhab 2.5.x, but although it ran for a few days, it made a lot of problems after a restart. I already started porting it to a real 2.x binding, but due to the current situation I have little time right atm - I hope I can work on this soon.

**Comment by musthafapa** [comment]
Thanks for the reply 

> By default, all that communication goes to the coordinator, which is connected to my openhab server. 

This was my question. is this a xbee radio connected to Raspberry Pi ?

**Comment by juri8** [comment]
right, the xbee coordinator is connected to a raspberry that also runs my openhab server. it's simply connected via a USB adapter.

**Comment by musthafapa** [comment]
Great. I understand XBee + Arduino via USB to Raspberry Pi.
